files/etc/uci-defaults/29-ns-storage-default

```diff
@@ -0,0 +1,13 @@
+. /lib/functions.sh
+. /lib/upgrade/common.sh
+
+export_bootdevice && export_partdevice diskdev 0
+if readlink "/sys/block/$diskdev" | grep -q "usb"; then
+    exit 1
+else
+    [ "$(uci -q get fstab.ns_data)" = "mount" ] && exit 0
+    jdev=$(/usr/libexec/ns-storage-setup-partition)
```

Comment:
If the setup-partition exists with 1, the script should terminate to avoid:
- calling add-storage on an empty variable `$part`
- outputting an error like `Failed to parse json data: unexpected end of data`